services/wallet/router/fees/estimated_time.go

```diff
@@ -215,8 +233,8 @@ func (f *FeeManager) estimatedTimeV2(feeHistory *FeeHistory, txMaxFeePerGas *big
 	}
 
 	for _, p := range inclusions {
-		baseFeePercentileIndex := getBaseFeePercentileIndex(sortedBaseFees, p.baseFeePercentile, networkCongestion)
-		if txBaseFee.Cmp(sortedBaseFees[baseFeePercentileIndex]) >= 0 && txPriorityFee.Cmp(p.priorityFee) >= 0 {
+		baseFeePercentileIndex := getBaseFeePercentileIndex(uniqueBaseFees, p.baseFeePercentile, networkCongestion)
+		if txBaseFee.Cmp(uniqueBaseFees[baseFeePercentileIndex]) >= 0 && txPriorityFee.Cmp(p.priorityFee) >= 0 {
```

Comment:
I am not sure if we should check this `txPriorityFee.Cmp(p.priorityFee) >= 0` in this condition?
Or should we check it for all chains?

Reply:
I've updated this condition to check for priority fees only if estimating time for L1 chains.